src/views/login.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react'
+import Buttons from '../components/buttons';
+
+export default class login extends Component {
```

Comment:
It is good practice for class names to begin with a capital letter. Please rename `login` to `Login`

Reply:
Thanks, let me work on this.